governance/votes/JanSky Executive Exercise June 12, 2023.md

```diff
@@ -0,0 +1,168 @@
+| QGov        | [0xB0524D8707F76c681901b782372EbeD2d4bA28a6](https://etherscan.io/address/0xB0524D8707F76c681901b782372EbeD2d4bA28a6) | 29.76 |
+| TRUE NAME   | [0x612f7924c367575a0edf21333d96b15f1b345a5d](https://etherscan.io/address/0x612f7924c367575a0edf21333d96b15f1b345a5d) | 29.76 |
+| UPMaker     | [0xbb819df169670dc71a16f58f55956fe642cc6bcd](https://etherscan.io/address/0xbb819df169670dc71a16f58f55956fe642cc6bcd) | 29.76 |
+| vigilant    | [0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61](https://etherscan.io/address/0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61) | 29.76 |
+| WBC         | [0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47](https://etherscan.io/address/0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47) | 20.16 |
+| PBG         | [0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2](https://etherscan.io/address/0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2) | 9.92  |
+| Bandhar     | [0xE83B6a503A94a5b764CCF00667689B3a522ABc21](https://etherscan.io/address/0xE83B6a503A94a5b764CCF00667689B3a522ABc21) | 7.68  |
+| Libertas    | [0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f](https://etherscan.io/address/0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f) | 7.04  |
+| PALC        | [0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A](https://etherscan.io/address/0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A) | 2.24  |
+| Harmony     | [0xF4704Aa4Ad22cAA2A3Dd7A7C529B4C32f7A421F2](https://etherscan.io/address/0xF4704Aa4Ad22cAA2A3Dd7A7C529B4C32f7A421F2) | 1.92  |
+| VoteWizard  | [0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1](https://etherscan.io/address/0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1) | 1.6   |
+| Navigator   | [0x11406a9CC2e37425F15f920F494A51133ac93072](https://etherscan.io/address/0x11406a9CC2e37425F15f920F494A51133ac93072) | 0.32  |
+
+To learn more about Delegate Compensation, please read this [section](https://mips.makerdao.com/mips/details/MIP101#2-6-3-1-1-the-process-for-modifying-the-number-of-pd-and-rd-slots-must-generally-aim-to-have-changes-in-the-number-of-slots-happen-at-the-end-of-the-election-season-but-in-extraordinary-circumstances-which-must-be-specified-by-the-governance-scope-they-can-be-done-with-1-month-warning-) of the Atlas.
+
+### Andromeda Output Conduit
+
+As per this forum [post](https://forum.makerdao.com/t/rwa015-project-andromeda-technical-assessment/20974/11) a new Output Conduit will be activated to allow Andromeda (RWA0151-A) to choose between Maker [PSMs](https://manual.makerdao.com/module-index/module-psm#trade-offs), if this executive proposal passes. 
+
+Code for the MultiSwapOutputConduit can be found [here]([https://github.com/makerdao/rwa-toolkit/blob/8d30ed2cb657641253d45b57c894613e26b4ae1b/audits/ChainSecurity_MakerDAO_RWA_Toolkit_audit.pdf).
+
+### DssCron Chainlog Housekeeping
+
+As per this forum [post](https://forum.makerdao.com/t/dsscron-housekeeping-additions/21292) the Chainlog will be brought up to date with past DssCron addresses and ones required by the new additions in this spell, if this executive proposal passes. 
```

Comment:
Yeah, feels fine. I don't think you need to list the addresses in the spell, though I also don't think its an issue if you do.

Reply:
I would maybe be a little more clear in your phrasing. 'Addresses relating to DssCron and the keeper network will be added to the chainlog.' You can also link the chainlog, for those that don't know what it is.